src/main/java/use_cases/PostManager.java

```diff
@@ -66,39 +66,57 @@ public void createPost(User owner, LocalDateTime postedTime, Recipe recipe, Stri
 
     /**
      * Comments a post iff could be done returns true
-     * @param post
-     * @param comment
-     * @return
+     * @param post: The post object itself
+     * @param comment: The comment is being added
+     * @return true iff it could be added the comment
      */
     public boolean commentPost(Post post, Comment comment){
         return interactPost(post, comment, true);
     }
 
     /**
      *  likes a post and saves the state
-     * @param post
-     * @param user
-     * @return
+     * @param post: the post is being liked
+     * @param user: the user that is giving the like
+     * @return true if the post could be liked
      */
     public boolean likePost(Post post, User user) {
         return interactPost(post, user, false);
     }
 
     /**
      * Adds a post directly from object
-     * @param post
+     * @param post: the post object is being added
      */
     public void createPost(Post post){
```

Comment:
I'm not sure if createPost is the best method name here. If you think about it, we've already created our Post (we're passing in the Post object). We're just adding the Post to the system (PostManager), and we should also update the DB here. However, I can add the DB call if you'd like (since I'll implement the Post related DB methods)

Reply:
Actually could you please edit this method because PostController calls postManager.createPost(currUser, timeNow, recipe, category); when making a new post. createPost should take in those parameters to make the post.

Reply:
And could you also save the post into the database in this method please?

Reply:
In the current main version it says createPost looks like this:
    public void createPost(User owner, LocalDateTime postedTime, Recipe recipe, String category)
    {
        Post p = new Post(owner.getId(), postedTime, recipe, category);
        this.addPostToList(p);
    }

did you change this by any chance?